Models/Project/DonutShop.cs

```diff
@@ -8,6 +8,6 @@ public class DonutShop
     public int DonutShopId { get; set; }
     public string DonutShopName { get; set; }
     public string DonutShopAddress { get; set; }
+    public string DonutShopCityState { get; set; }
     public string DonutShopWebsite { get; set; }
-    public string DonutShopImage { get; set; }
```

Comment:
Did we decide to take the image out?

Reply:
yes there was no easy way to get individual images